ServerData.java

```diff
@@ -0,0 +1,1114 @@
+     * The point40
+     */
+    public void setPoint40(List<Double> point40) {
+        this.point40 = point40;
+    }
+
+    /**
+     *
+     * @return
+     * The point41
+     */
+    public List<Double> getPoint41() {
+        return point41;
+    }
+
+    /**
+     *
+     * @param point41
+     * The point41
+     */
+    public void setPoint41(List<Double> point41) {
+        this.point41 = point41;
+    }
+
+    /**
+     *
+     * @return
+     * The point42
+     */
+    public List<Double> getPoint42() {
+        return point42;
+    }
+
+    /**
+     *
+     * @param point42
+     * The point42
+     */
+    public void setPoint42(List<Double> point42) {
+        this.point42 = point42;
+    }
+
+    /**
+     *
+     * @return
+     * The point43
+     */
+    public List<Double> getPoint43() {
+        return point43;
+    }
+
+    /**
+     *
+     * @param point43
+     * The point43
+     */
+    public void setPoint43(List<Double> point43) {
+        this.point43 = point43;
+    }
+
+    /**
+     *
+     * @return
+     * The point44
+     */
+    public List<Double> getPoint44() {
+        return point44;
+    }
+
+    /**
+     *
+     * @param point44
+     * The point44
+     */
+    public void setPoint44(List<Double> point44) {
+        this.point44 = point44;
+    }
+
+    /**
+     *
+     * @return
+     * The point45
+     */
+    public List<Double> getPoint45() {
+        return point45;
+    }
+
+    /**
+     *
+     * @param point45
+     * The point45
+     */
+    public void setPoint45(List<Double> point45) {
+        this.point45 = point45;
+    }
+
+    /**
+     *
+     * @return
+     * The point46
+     */
+    public List<Double> getPoint46() {
+        return point46;
+    }
+
+    /**
+     *
+     * @param point46
+     * The point46
+     */
+    public void setPoint46(List<Double> point46) {
+        this.point46 = point46;
+    }
+
+    /**
+     *
+     * @return
+     * The point47
+     */
+    public List<Double> getPoint47() {
+        return point47;
+    }
+
+    /**
+     *
+     * @param point47
+     * The point47
+     */
+    public void setPoint47(List<Double> point47) {
+        this.point47 = point47;
+    }
+
+    /**
+     *
+     * @return
+     * The point48
+     */
+    public List<Double> getPoint48() {
+        return point48;
+    }
+
+    /**
+     *
+     * @param point48
+     * The point48
+     */
+    public void setPoint48(List<Double> point48) {
+        this.point48 = point48;
+    }
+
+    /**
+     *
+     * @return
+     * The point49
+     */
+    public List<Double> getPoint49() {
+        return point49;
+    }
+
+    /**
+     *
+     * @param point49
+     * The point49
+     */
+    public void setPoint49(List<Double> point49) {
+        this.point49 = point49;
+    }
+
+    /**
+     *
+     * @return
+     * The point50
+     */
+    public List<Double> getPoint50() {
+        return point50;
+    }
+
+    /**
+     *
+     * @param point50

```

Comment:
Your understanding about GET/POST is incorrect. Please read up on how Requests/Responses work. You will have total of 2 requests:
1- Get list of 50 points- Request parameters: None, Response: List of points
2- Get shortest path from A to B- Request parameters: A,B, Response: Shortest path
Both of them can be GET requests and you don't need a POST request as such.